backend/src/main/java/com/redhat/cloud/notifications/db/EndpointResources.java

```diff
@@ -228,30 +228,30 @@ public boolean updateEndpoint(Endpoint endpoint) {
             return false;
         } else if (endpoint.getProperties() == null) {
             return true;
-        } else {
```

Comment:
Having `else` blocks in the code even if they could be omitted makes the code easier to read/review/modify, especially for someone who didn't write it.

Reply:
imo every additional indentation makes code harder to read

Reply:
I'm +1 on keeping the else.

Below chunk is not trivial and is better to have it surronded in a block.

Now... if you would like to wrap below code into a function i wouldn't mind a `return the_new_function()`